integration-cli/docker_cli_network_unix_test.go

```diff
@@ -1008,3 +1008,41 @@ func verifyIPAddresses(c *check.C, cName, nwname, ipv4, ipv6 string) {
 	out, _ = dockerCmd(c, "inspect", fmt.Sprintf("--format='{{ .NetworkSettings.Networks.%s.GlobalIPv6Address }}'", nwname), cName)
 	c.Assert(strings.TrimSpace(out), check.Equals, ipv6)
 }
+
+func (s *DockerSuite) TestUserDefinedNetworkConnectDisconnectAlias(c *check.C) {
+	testRequires(c, DaemonIsLinux, NotUserNamespace)
+	dockerCmd(c, "network", "create", "-d", "bridge", "aliasnet1")
```

Comment:
i find the name `aliasnet1` to be confusing, just call it `net1` unless I'm missing something. Same for aliasnet2